src/components/common/ui/link-mui/Link.styles.ts

```diff
@@ -0,0 +1,25 @@
+import { SxProps, Theme } from '@mui/material/styles';
+
+import palette from '@/styles/theme/constants/pallete';
+import typography from '@/styles/theme/constants/typography';
+
+export const LinkStyles = (type): SxProps<Theme> => ({
+  color: type == 'white' ? palette.grey[800] : palette.info[400],
+  typography: {
```

Comment:
As I know, body1 is the default typography so you don't need to write it in styles.
Try it and if I'm wrong, then write typography like `mobile: 'body1'` instead of `typography.`